pkg/cri/sbserver/helpers.go

```diff
@@ -84,10 +84,6 @@ const (
 	containerKindSandbox = "sandbox"
 	// containerKindContainer is a label value indicating container is application container
 	containerKindContainer = "container"
-	// imageLabelKey is the label key indicating the image is managed by cri plugin.
-	imageLabelKey = criContainerdPrefix + ".image"
-	// imageLabelValue is the label value indicating the image is managed by cri plugin.
-	imageLabelValue = "managed"
```

Comment:
@mikebrow we can avoid it for `pkg/cri/sbserver` for now, but for `pkg/cri/server` we have to move in order to avoid cyclic dependency with `pkg/cri/store` , since i already extracted for `pkg/cri/server` i have used the extracted ones, if won't remove it from here and use these for sb server?